source/common/tcp/original_conn_pool.cc

```diff
@@ -176,7 +178,8 @@ void ConnPoolImpl::onConnectionEvent(ActiveConn& conn, Network::ConnectionEvent
         reason = ConnectionPool::PoolFailureReason::LocalConnectionFailure;
       }
 
-      std::list<PendingRequestPtr> pending_requests_to_purge(std::move(pending_requests_));
+      std::list<PendingRequestPtr> pending_requests_to_purge;
```

Comment:
any specific reason for this change?

Reply:
clang tidy failed with the std::move because it's accessed below and thou shalt not access something which has been std::moved.

Reply:
Makes sense, thanks for explaining!